Python/pywarpx/picmi.py

```diff
@@ -458,6 +464,20 @@ def initialize_inputs(self, species_number, layout, species, density_scale):
 
 
 class AnalyticDistribution(picmistandard.PICMI_AnalyticDistribution, DensityDistributionBase):
+    """
```

Comment:
Note the once PR #107 from PICMI is merged, this code can be removed since the WarpX specific argument is no longer needed.